### DISCUSSION TITLE
Infrastructure Week - Day 5

### DISCUSSION THREAD

**Discussion by the-maldridge** [void-linux/void-packages#45165] [General]
Feel free to ask followup questions in this thread about [the Friday post of infrastructure week.](https://voidlinux.org/news/2023/07/infra-week-day-5.html)

**Comment by pfpulux** [comment]
Nice post, thank you.
Can you explain the NetAuth-setup more in detail also in compare to e.g. to KeyCloak.

**Comment by module0x90** [comment]
I enjoyed reading this, great post.
Thank you for taking the time to write this up. Very insightful.
